data/pokemon_rule_output.txt

```diff
@@ -0,0 +1,147 @@
+
+
+
+-------Primary Color-------
+['hasPrimaryColor Green', 'hasPrimaryColor Red', 'hasPrimaryColor Blue', 'hasPrimaryColor White', 'hasPrimaryColor Brown', 'hasPrimaryColor Yellow', 'hasPrimaryColor Purple', 'hasPrimaryColor Pink', 'hasPrimaryColor Gray', 'hasPrimaryColor Black']
+
+
+
+-------Shape-------
+['hasShape Quadruped', 'hasShape Upright', 'hasShape Armor', 'hasShape Squiggle', 'hasShape Bug-Wings', 'hasShape Wings', 'hasShape Humanoid', 'hasShape Legs', 'hasShape Blob', 'hasShape Heads', 'hasShape Tentacles', 'hasShape Arms', 'hasShape Fish', 'hasShape Ball']
+
+
+
+normal_attack_effectiveness
+--- how effective againstNormal ---
```

Comment:
It's actually how effective normal is against this pokemon. 
The predicate name "howEffectiveNormal" is fine though.